client/plots/matrix.interactivity.js

```diff
@@ -720,6 +720,7 @@ function setTermActions(self) {
 		const sandbox = newSandboxDiv(self.opts.plotDiv || select(self.opts.holder.node().parentNode))
 		sandbox.header.text(t.tw.term.name)
 		const arg = {
+			debugmode: self.app.opts.state.debug,
```

Comment:
please see if it's reliable to access debug flag this way

Reply:
should be `debugmode: self.app.opts.debug`. Debugmode should not be saved as part of state, it's really environment dependent (dev/test versus prod), we don't want a prebuilt session state to expose debugmode in prod, only in dev/test.

Reply:
yes agree. i changed it from state to `opts{}`, and reverted such in url.html
i will create new issue to set app.opts.debug with environment-level setting